### DISCUSSION TITLE
SD card issue on custom board

### DISCUSSION THREAD

**Discussion by OrlyShdema** [polarfire-soc/polarfire-soc-documentation#339] [General]
Hi,
We placed a mictoSD card in our target board, and there is no eMMC.
The MSS configurator configure to SD enable and 3.3v.
The microSD is recognized by the HSS, and working properly under uboot and even boot the Linux from it.
 
![image](https://github.com/polarfire-soc/polarfire-soc-documentation/assets/75666391/a1e47df1-3188-4acc-a183-99904486e719)

 

The microSD information from uboot:

 
![image](https://github.com/polarfire-soc/polarfire-soc-documentation/assets/75666391/8b018381-f063-4df7-98ac-d31cc38a77a3)

The microSD prepared from the compilation wic, so all the partitions available:
![image](https://github.com/polarfire-soc/polarfire-soc-documentation/assets/75666391/e12040e2-fb4e-4e5b-99f8-e11b658b4028)

 

The issue is that something in the Linux (or configuration) cause microSD card to "stuck":
![image](https://github.com/polarfire-soc/polarfire-soc-documentation/assets/75666391/e74d8c91-8a44-4991-94c1-eae0082b1468)

 

After HSS reset command, the microSD doesn't recognized:
![image](https://github.com/polarfire-soc/polarfire-soc-documentation/assets/75666391/bcd8ccb0-efa6-4043-b381-07491f53e311)

 

We started from icicle 2023.02 project, DTS with changes and boot log attached. 
The same standard microSD (SanDisk Ultra 16GB HC1, 10) with adapter works fine in the icicle.

pls. advise
Thanks,
Orly

**Comment by hughbreslin** [comment]
Hi @OrlyShdema are you using the same voltage configuration for the custom board as you are on Icicle? i.e the bank 4 reference and aux voltages? It could cause an issue with the initialization if there is a mismatch there
